Source/bmalloc/libpas/src/libpas/pas_darwin_spi.h

```diff
@@ -31,9 +31,16 @@
 
 #if PAS_OS(DARWIN)
 #if defined(__has_include) && __has_include(<pthread/private.h>)
+
+// Swift Interop does not allow headers inside extern "C".
```

Comment:
This is kind of mysterious. Can we make it less mysterious? What specifically is problematic about including a header inside an extern C block?

Reply:
One way to be less mysterious is to pair this with a Radar and a workaround comment.